_posts/2019-08-13-how-to-deploy-production-grade-vpc-aws.adoc

```diff
@@ -683,25 +684,29 @@ provider "aws" {
 }
 
 terraform {
+  # TODO: this didn't work for me. after creating the region/account/common/terragrunt hcl files, it complained that
+  # backend was defined more than once.
   # The configuration for this backend will be filled in by Terragrunt or via a backend.hcl file. See
   # https://www.terraform.io/docs/backends/config.html#partial-configuration
   backend "s3" {}
 
+  # TODO: bump this to a higher version
   # Only allow this Terraform version. Note that if you upgrade to a newer version, Terraform won't allow you to use an
   # older version, so when you upgrade, you should upgrade everyone on your team and your CI servers all at once.
   required_version = "= 0.12.6"
 }
 ----
 
+# TODO: We say to replace version here, but we didn't give that advice in the Landing Zone guides. Should add it there.
```

Comment:
Actually, we should fix it so that this is pinned to a version (meaning, we should prefer the approach in the LZ article). This ensures that the user is using version locked code, which makes this guide accurate and workable even if we publish backwards compatible changes to the underlying modules.

Reply:
The best solution of all is to use AsciiDoc `include` and pull these code snippets from real, tested code in `aws-service-catalog` or `aws-architecture-catalog`. That was the original plan and one of the main reasons to se AsciiDoc for these guides. If we do that, then as long as our examples pass tests, the code in these guides will work.

Reply:
Yes, yes, yes. I can't +1 that enough.

Reply:
- [ ] Use AsciiDoc includes, yo!

Reply:
I want to do this, but in the interest of time, getting this out the door sooner, going to table this.